man/systemd.exec.xml

```diff
@@ -225,6 +225,53 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ProcMode=</varname></term>
+
+        <listitem><para>Takes one of <literal>noaccess</literal>, <literal>invisible</literal>,
+        <literal>ptraceable</literal> or <literal>default</literal>. Defaults to the latter. This controls
+        the <literal>hidepid=</literal> mount option of the <literal>procfs</literal> instance for the unit:
```

Comment:
I think we should have a new sentence starting with "When set ...".

Reply:
Looking at this more, "hidepid=foo" means that foo will be shown ;( The usual wisdom about negative options applies.

Please add a sentence that explains what the kernel option does: "the hidepid= mount option of procfs that controls which directories with process metainformation (`/proc/<replaceable>PID</>/`) are visible." or something like that.

Reply:
I didn't like the negative option mess, but I didn't want to depart too widely from the kernel's native option names. hence i just accepted our fate on this, even though I renamed the setting itself, because "hidepid=" is just too crappy a name...